src/resource.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
```

Comment:
I wonder if this file should live in `lib/` instead of `src/`, and if the file should be renamed to `kubernetes_resources.py`